week-5/A-fetch/exercise.js

```diff
@@ -17,10 +17,33 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
-    .then(function(response) {
-        return response.text();
-    })
-    .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+var btn = document.querySelector("#btn");
+var clear = document.querySelector("#clear");
+var greetText = document.querySelector("#greeting-text");
+
+var count = 0;
+
+btn.addEventListener("click", function() {
+	fetch("https://codeyourfuture.herokuapp.com/api/greetings")
+		.then(function(response) {
+			return response.text();
+		})
+		.then(function(greeting) {
+            // Write the code to display the greeting text here
+            var box = document.querySelector("#greeting-text");
+			var paragraph = document.createElement("p");
+            paragraph.innerText = greeting;
+            count++;
+            if(count % 2 === 0){
+                paragraph.style.color = "blue";
+            } else {
+                paragraph.style.color = "red";
+            }
+            
+			box.appendChild(paragraph); 
+		});
```

Comment:
Try to keep formatting and indentation consistent. This event listener code is hard to read because of the formatting. Also, why did you decide to change the `p` to a `div` and add the `p` element each time?

Reply:
I do not know where you see that. I never did change the `p` to a `div`. I just created a `p` element and put inside that `div` (box). So that all paragraphs will appear in the box.

Reply:
It looks like the [HTML was changed here](https://github.com/CodeYourFuture/js-exercises/pull/53/files#diff-01e4e272c3c492e60c22a4341b688359L24).